docs/TuxMutableClass.md

```diff
@@ -0,0 +1,188 @@
+
+Let's take a look at an example: [#](#create-mutable-class-example)
+<a id="create-mutable-class-example"></a>
+
+```javascript
+  var Message = React.createMutableClass({
+    // the traits we want to watch to see if a re-render is required
+    mutableTraits: {
+      props: ['text', 'timestamp'],
+      state: 'editing'
+    },
+
+    render: function () {
+      return (
+        <div>
+          <h1>{this.props.message.text}</h1>
+        </div>
+      );
+    }
+  });
+```
+
+Below is the api documentation for this method.
+
+### <a id="React-createMutableClass"></a>React.createMutableClass [#](#React-createMutableClass)
+
+```javascript
+  var Message = React.createMutableClass(mutableClassProps);
+```
+
+#### <a id="createMutableClass-mutableClassProps"></a>Parameter - `mutableClassProps` - type: OBJECT - required [#](#createMutableClass-mutableClassProps)
+Methods and properties to be extended onto the `mutableClass` object returned. Expected keys:
+
+##### <a id="mutableClassProps-mutableTraits"></a>Property - `mutableClassProps.mutableTraits` - type: OBJECT - optional [#](#mutableClassProps-mutableTraits)
+The traits that will be registered onto the mutable class.
+
+**NOTE: If the `mutableTraits` property is not included in the `mutableClassProps` object, instead of applying the TuxMutableRenderMixin to the return object, the PureRenderMixin will be applied instead. The PureRenderMixin implements `shouldComponentUpdate`, but it only does a shallow compare of the top-most keys in props and state.**
+
+Expected keys:
+
+###### <a id="mutableTraits-props"></a>Property - `mutableTraits.props` - type: STRING, ARRAY of STRINGs, ARRAY of ARRAYs of STRINGs - optional [#](#mutableTraits-props)
+A trait, or array of traits, or an array of arrays of traits, within the components props object to create a path for. An example will follow the `mutableTraits.state` explanation.
+
+###### <a id="mutableTraits-state"></a>Property - `mutableTraits.state` - type: STRING, ARRAY of STRINGs, ARRAY of ARRAYs of STRINGs - optional [#](#mutableTraits-state)
+A trait, or array of traits, or an array of arrays of traits, within the components state object to create a path for.
+
+```javascript
+    var mutableClassProps = {};
+    mutableClassProps.mutableTraits = {
+      props: 'text',
+      state: 'editing'
+    };
+
+    // OR
+    mutableClassProps.mutableTraits = {
+      props: ['text', 'timestamp'],
+      state: 'editing'
+    };
+
+    // OR it is possible to specify the hierarchy of keys to search through to improve performance of the one time deep search/provide greater specificity regarding the required property
+    mutableClassProps.mutableTraits = {
+      props: [
+        ['message', 'text'],
+        ['message', 'timestamp']
+      ],
+      state: 'editing'
+    };
+```
+
+#### <a id="mutableClassProps-render"></a>Parameter - `mutableClassProps.render` - type: FUNCTION - required [#](#mutableClassProps-render)
```

Comment:
Do you wanna copy and paste over the section I have in the TuxOwnerClass doc about the `React` lifecycle methods and specs?